examples/gpu/polymer_sync/1_6-31g_inp_gpu.py

```diff
@@ -0,0 +1,57 @@
+from mrh.my_pyscf.gpu import libgpu
+
+import pyscf
+from gpu4pyscf import patch_pyscf
+
+from mrh.tests.gpu.geometry_generator import generator
+from pyscf import gto, scf, tools, mcscf
+from mrh.my_pyscf.mcscf.lasscf_o0 import LASSCF
+from pyscf.mcscf import avas
+
+from pyscf import lib
+lib.logger.TIME_LEVEL = lib.logger.INFO
+
+# -- this should all be inside constructor
```

Comment:
Don't say "should be". It either is or it isn't. Remember that this is a sample input file that users will read to try to understand how to use the code. Be precise and unambiguous.

Reply:
@valay1 replace all of those sections with the simpler call to gpu = libgpu_init(). Behavior should be identical now with the commits from yesterday.

Similarly, remove the comments around libgpu_destroy_device() at bottom of input files.